frontend/src/pages/index.tsx

```diff
@@ -39,7 +39,7 @@ const Home: NextPage = () => {
 };
 
 // Change this to getStaticProps if you're not using server-side rendering
```

Comment:
nit: update the comment for if/when we want to change back to getserversideprops

Reply:
Good call, thanks.